ydb/public/lib/ydb_cli/commands/ydb_tools.cpp

```diff
@@ -225,6 +225,12 @@ void TCommandRestore::Config(TConfig& config) {
         " instead of silently skipping its removal.")
         .StoreTrue(&VerifyExistence);
 
+    config.Opts->AddLongOption("retries", "Max retry count for every request.")
+        .DefaultValue(10)
+        .Handler([this](const TString& arg) {
+            Retries = FromString<ui32>(arg);
+        });
```

Comment:
Почему так сложно, а не:
```suggestion
    config.Opts->AddLongOption("retries", "Max retry count for every request.")
        .DefaultValue(10).StoreResult(&Retries);
```
?